tests/e2e/cmd/config.yml

```diff
@@ -0,0 +1,116 @@
+version: 1
+
+accounts:
+  - name: faucet
+    coins: ["99898000tkx"]
+  - name: val1
+    coins: ["101000tkx"]
+  - name: reward-pool-admin
+    # titan1up5lnguhgpu2qdqz9hav34yyvs0kjw3pruzrh7
+    coins: ["1000tkx"]
+    mnemonic: throw beef member notable devote begin extend possible vibrant original point table
+
+validators:
+  - name: val1
+    bonded: 100000tkx
+    gentx:
+      min-self-delegation: "5000000000000000000"
+    app:
+      minimum-gas-prices: 0atkx
+      api.enable: true
+      api.swagger: true
+      api.enabled-unsafe-cors: true
+    config:
+      moniker: "titan val1"
+      consensus:
+        create_empty_blocks: false
+        create_empty_blocks_interval: 30s
+        timeout_propose: 3s
+        timeout_propose_delta: 0.5s
+        timeout_prevote: 1s
+        timeout_prevote_delta: 0.5s
+        timeout_precommit: 1s
+        timeout_precommit_delta: 0.5s
+        timeout_commit: 1s
+      rpc:
+        timeout_broadcast_tx_commit: 10s
+    client:
+      chain-id: "titan_18888-1"
```

Comment:
- Please change chain id to `titan_18887-1`. we will use chain id lower than 18888 for testnet local